doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -390,6 +390,20 @@ A simple self test callback is shown below for illustrative purposes.
     return ret;
   }
 
+=head1 NOTES
+
+The FIPS provider in OpenSSL 3.1 includes some non-FIPS validated algorithms,
+consequently the property query C<fips=yes> is mandatory for applications that
+want to operate in a FIPS approved manner.  The algoriths are:
+
+=over 4
+
+=item Triple DES
+
+=item EdDSA
```

Comment:
Maybe we could add a note that future validated versions might make this algorithm approved so people can be prepared in advance? Or do we want to keep EdDSA as non-approved even if we revalidate 3.1.x sometime in future?

Reply:
Adding that note will provoke questions as to when, so I'd prefer not to.  We've no idea when FIPS 186-5 will become canon or how long a followup validation would take.

Changing the documentation when we're resubmitting at some point after 186-5 is done seems like a reasonable route to me.